src/App.scss

```diff
@@ -1,70 +1,110 @@
+@mixin pages {
+  display: flex;
+  justify-content: center;
+  margin-top: 10px;
+  background-color: white;
+}
+html {
+  background-color: grey;
+}
+h1 {
+  color: white;
+}
+datalist{
+  display:none;
+}
+img {
+  width: 22vw;
+}
 .App {
   text-align: center;
 }
-
-.filmContainer {
-  display: flex;
+.header {
+  background-color: transparent;
+}
+.main-page-film-container {
+  @include pages();
   flex-wrap: wrap;
-  justify-content: center;
 
   div {
     margin: 10px;
     width: 25vw;
     height: 100%;
   }
 }
-img {
-  width: 25vw;
+.film-page-film-container {
+  @include pages();
+  flex-direction: row;
+  * {
+    margin: 10px;
+  }
+  .film-info {
+    text-align: left;
+    margin: 10px;
+    width: 35vw;
+  }
 }
-
-
-.mini_form *{
-display: inline;
-font-size: 1.3rem;
-border-radius: 5px;
-
+.mini-form * {
+  display: inline;
+  font-size: 1.3rem;
+  border-radius: 5px;
+}
+.input-container {
+  background-color: white;
+  border: solid black;
 }
 
-.input[placeholder]{
-
-  padding: 0% 1%;
-  width: 30vw;
-  border:none;
-
+.input[placeholder] {
+  width: 35vw;
+  padding: 0 1%;
+  background-color: transparent;
+  border: none;
 }
-.input:focus{
+.input:focus {
   outline: none;
 }
-
-.home_button{
+.home-button {
   display: inline-block;
   background-color: rgb(121, 121, 209);
   color: white;
+  line-height: 1.7rem;
+  border: solid black;
 }
-.home_button:hover{
+.home-button:hover {
```

Comment:
we can use nesting to make it more structured: .home-button { someProp: someVal; &: hover { .... } }